samples/apps/smallbank/app/amalgamate.fbs

```diff
@@ -0,0 +1,10 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.	
+// Licensed under the Apache 2.0 License.
+
+table Amalgamate
+{
+    name_src:uint;
```

Comment:
Should these be `uint64`? The other ID fields are.

Reply:
Probably, I was just mimicking the way there were defined in the `small_bank_client` and `smallbank.cpp` files